code/obj/item/playing_cards.dm

```diff
@@ -1,1137 +1,1265 @@
+        else
+            atk = chosen_card_type.ATK
+            def = chosen_card_type.DEF
+        name += " [atk]/[def]"
+        desc = chosen_card_type.card_data
+        desc += " ATK [atk] | DEF [def]"
+        icon_state = "stg-general-[rand(1,NUMBER_GENERAL)]"
+
+    proc/stg_effect(var/list/possible_card_types)
+        var/path = pick(possible_card_types)
+        var/datum/playing_card/griffening/effect/chosen_card_type = new path
+
+        name = chosen_card_type.card_name
+        desc = chosen_card_type.card_data
+        icon_state = "stg-general-[rand(1,NUMBER_GENERAL)]"
+
+    proc/stg_area(var/list/possible_card_types)
+        var/path = pick(possible_card_types)
+        var/datum/playing_card/griffening/area/chosen_card_type = new path
+
+        name = chosen_card_type.card_name
+        desc = chosen_card_type.card_data
+        icon_state = "stg-general-[rand(1,NUMBER_GENERAL)]"
+
+    proc/add_foil() //makes the card shiiiiiny
+        UpdateOverlays(image(icon,"stg-foil"),"foil")
+        foiled = TRUE
+        name = "foil [name]"
+
+/obj/item/playing_card/expensive //(¬‿¬)
+    desc = "Tap this card and sacrifice one Yourself to win target game."
+    icon_state = "stg-general-0"
+    var/list/prefix1 = list("Incredibly", "Strange", "Mysterious", "Suspicious", "Scary")
+    var/list/prefix2 = list("Rare", "Black", "Dark", "Shadowy", "Expensive", "Fun", "Gamer")
+    var/list/names = list("Flower", "Blossom", "Tulip", "Daisy")
+
+    New()
+        ..()
+        name = "[pick(prefix1)] [pick(prefix2)] [pick(names)]"
+        update_stored_info()
+
+    MouseDrop(var/atom/target as obj|mob)
+        ..()
+        if(tapped)
+            var/mob/user = usr
+            user.deathConfetti()
+            playsound(get_turf(user.loc), 'sound/musical_instruments/Bikehorn_1.ogg', 50)
+            user.visible_message("<span class='combat'><b>[uppertext(user.name)] WINS THE GAME!</b></span>")
+            if(!foiled)
+                user.take_brain_damage(1000)
+            else
+                user.partygib(1)
+
+/obj/item/card_group //since "playing_card"s are singular cards, card_groups handling groups of playing_cards in the form of either a deck or hand 
+    name = "deck of cards"
+    icon = 'icons/obj/items/playing_card.dmi'
+    dir = NORTH
+    w_class = 1.0
+    burn_point = 220
+    burn_output = 900
+    burn_possible = 2
+    health = 10
+    inventory_counter_enabled = 1
+    var/card_name //same function as playing_card card name
+    var/card_style = "plain" //the type of card back used for this group (references icon_state names in the dmi)
+    var/total_cards //how many cards are in a fully built deck of this type? (54 for plain decks, 78 for tarot, etc.) : used for reference on stack heights
+    var/is_hand = FALSE
+    var/max_hand_size = 18 //the number of cards you can have in a hand before it automatically becomes a deck
+    contextLayout = new /datum/contextLayout/instrumental(16)
```

Comment:
instrumental? 🤔

Reply:
it displays the buttons in a nice line over the player's head using the same format as instrument code (i.e. saxophone buttons)